public/ACMW/css/events.css

```diff
@@ -51,7 +42,7 @@ body {
 	color: #d5d5d5;
 }
 .navbar-nav li a:hover {
-	color: #fff;
+	color: #fff !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)